src/prefect/environments/storage/_healthcheck.py

```diff
@@ -117,8 +122,16 @@ def environment_dependency_check(flows: list):
     flow_file_path, python_version = sys.argv[1:3]
 
     print("Beginning health checks...")
+
+    flow_file_paths = ast.literal_eval(flow_file_path)
+
     system_check(python_version)
-    flows = cloudpickle_deserialization_check(flow_file_path)
+
+    if len(flow_file_paths) == 1 and ".py" in flow_file_paths[0]:
```

Comment:
Will there always be one flow file for file based storage?

Reply:
Thinking about it, it actually doesn't need to be a single flow file. Other storage options for file based only support a single file per storage option and I don't see a reason why docker couldn't still allow for multiple flows like it does right now. I will update this